content/guide/agents-cloudify-agent.md

```diff
@@ -0,0 +1,188 @@
+---
+
+title: Agent Configuration
+category: Agents
+publish: true
+abstract: "Description of the Cloudify Agent installation and configuration"
+pageord: 220
+
+autoscale_link: http://docs.celeryproject.org/en/latest/userguide/workers.html#autoscaling
+sc_link: https://technet.microsoft.com/en-us/library/bb490995.aspx
+---
+
+{{% gsSummary %}} {{% /gsSummary %}}
+
+# Installation Methods
+
+There are several methods by which an agent can be installed, configured and started.
+
+The `remote` method uses SSH on linux hosts and WinRM on windows hosts.
+The `init_script` and `provided` methods make use of an initialization script that gets executed on the host when it gets created; This usually takes place during the `create` operation of the cloud specific plugin used (e.g. in Openstack, `userdata` is used).
+
+To configure which installation method will be used, set the `install_method` property in the compute node `agent_config` property. For example:
+
+{{< gsHighlight  yaml  >}}
+node_templates:
+  my_vm:
+    type: cloudify.nodes.Compute
+    property:
+      agent_config:
+        # one of none, remote, init_script, provided
+        install_method: remote
+{{< /gsHighlight >}}
+
+Following are all the supported installation methods:
+
+* `none` - No agent will be installed on the host.
```

Comment:
Do you think it would be better to describe the prerequisites just under each mode listed here?